charter/index.html

```diff
@@ -302,14 +302,11 @@ <h3>
                                 <li>
                                     <a href="https://openid.net/connect/">OpenID Connect</a>
                                 </li>
-                                <li>
-                                    Access grants using <a href="https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html">W3C Verifiable Credentials</a>
-                                </li>
```

Comment:
Why remove this? Even if it had no prior incubation in the CG, it seems to makes sense. And as a matter of fact, @elf-pavlik did present something today during the CG call what involved VCs, IIUC...

Reply:
In the data interoperability panel (one of the CG panels), we look at using VCs to represent details of what access the end user delegated to the client (the app), we discussed it as part of the solution for https://github.com/solid/specification/issues/504 but going beyond the server side clients.

Reply:
@elf-pavlik  Then the WG homepage (https://beta.w3.org/groups/wg/vc/ or https://www.w3.org/2017/vc/WG/) would be more appropriate than the charter.